UI/Sources/ShoppingCart/Views/CartStepperView.swift

```diff
@@ -35,13 +35,13 @@ struct CartStepperView: View {
     @ViewBuilder
     var minusImage: some View {
         Image(systemName: itemModel.quantity == 1 ? "trash" : "minus")
-            .foregroundColor(.accentColor)
+            .foregroundColor(.projectPrimary())
```

Comment:
Don't we need the @Environment(\.project) here?

Reply:
42149bf95c5bcc1a82518cc5ba35d7ae5fe67014